src/commands.py

```diff
@@ -0,0 +1,157 @@
+last_card = None
+reset_users = ['Gerst','ace','Lerker','Shaper']
+obey_dict = {
+        'Shaper': 'I obey, Master Shaper.',
+        'ace': 'I obey, Admiral Ace.',
+        'muCephei': 'I obey, muCephei.',
+        'Gerst': 'I obey, Artificer Gerst.',
+        'Lerker': 'I obey, Great Lerker.',
+        'ShakeAndShimmy': 'I obey, Chancellor ShakeAndShimmy.',
+        'angelforge': 'I obey, Lord AngelForge.',
+        'JimWolfie': 'Suck my necrotic dick, Jim.',
+        'Skuloth': 'Zur is for scrubs, I refuse to obey.',
+        'Noon2Dusk': 'I obey, Inventor Noon.',
+        'razzliox': 'I obey, Razzberries.',
+        'ifarmpandas': 'Beep boop, pandas are the best.'
+}
+
+# ---------------------------
+# Command: Fetch
+# ---------------------------
+def cmd_fetch(message):
+    global last_card
+    response = ''
+    queries = re.findall(("<<([^<>]*)>>"), message.content.encode('utf-8'))
+
+    for s in queries:
+        query = s.encode('utf-8')
+        proc = subprocess.Popen(['mtg', query, '--json'], stdout=subprocess.PIPE)
+        result = str(proc.communicate()[0])
+
+        card_list = json.loads(result)
+
+        # Store the card if it's the only one
+        last_card = None
+        if len(queries) == 1 and len(card_list) == 1:
+            last_card = card_list[0]
+
+        # If no cards are found, we are done
+        if len(card_list) == 0:
+            response += '**' + query + '**: *The ritual summoned nothing but ash...*'
+            continue
+
+        # If an exact card is found, just print that one
+        # When you find the exact match, break out of the for card in cards loop
+        # Then "continue" the for s in queries to move to the next query
+        done = False
+        for card in card_list:
+            if (card['name'].encode('utf-8').lower() == query.lower()):
+                response += cards.get_card(message, card)
+                last_card = card
+                done = True
+                break
+        if done:
+            continue
+
+        # If more than 8 cards are found, don't spam chat
+        if len(card_list) > 8:
+            response += 'The incantations are too long; read them yourself'
+            continue
+
+        # Finally, if we've gotten to here, print all the cards
+        for card in card_list:
+            response += cards.get_card(message, card)
+
+    return response
+
+# ---------------------------
+# Command: Details
+# ---------------------------
+def cmd_details(message):
+    global last_card
+    if last_card is not None:
+        return cards.get_card_details(message, last_card)
+    else:
+        return 'You must divine a single entity first.'
+
+# ---------------------------
+# Command: Rulings
+# ---------------------------
+def cmd_rulings(message):
+    global last_card
+    if last_card is not None:
+        return cards.get_card_rulings(message, last_card)
+    else:
+        Y.send_message(message.channel, 'You must divine a single entity first.')
```

Comment:
Shouldn't this be 
return 'You must divine a single entity first.'

like it is above in details?